Marlin/src/module/tool_change.h

```diff
@@ -64,6 +63,8 @@
   #endif
 #endif
 
+
+extern float z_raise;
 /**
```

Comment:
Please pay attention to where blank lines are added. This should have no extra blank line in front, but should have a new blank line after. You're not doing these things just to troll me, are you? 🙄